pkg/rtcconfig/webrtc_config.go

```diff
@@ -190,6 +191,12 @@ func NewWebRTCConfig(rtcConf *RTCConfig, development bool) (*WebRTCConfig, error
 		}
 	}
 
+	net, err := stdnet.NewNet()
+	if err != nil {
+		return nil, err
+	}
+	s.SetNet(net)
```

Comment:
How does this work @paulwe ? I think this is used for doing network simulations. Were we creating the list for every participant/room?

Reply:
got it, Thank you Paul.